infra/gravity/cluster_install.go

```diff
@@ -190,10 +190,6 @@ func (c *TestContext) Upgrade(nodes []Gravity, installerURL, gravityURL, subdir
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	err = c.Status(nodes)
```

Comment:
I wonder why didn't you just move it before uploadInstalled above, but moved it into the upload function.

Reply:
I moved closer to the actual `upload upgrade` call - otherwise it would be two `Status` calls one after the other.